controller/controller_server/csi_controller_server.py

```diff
@@ -406,6 +406,7 @@ def CreateSnapshot(self, request, context):
             context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
             return csi_pb2.CreateSnapshotResponse()
 
+        pool = request.parameters.get(config.PARAMETERS_POOL)
```

Comment:
why not `pool_id`?

Reply:
because I copied the flow from create volume and tried to maintained the consistency

Reply:
i see. it might be the pool name on some of the other storage systems or something.